tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -29,7 +30,12 @@
     Page Should Contain    Model Serving
     Click Link      Model Serving
     Wait Until Page Contains    Manage and view the health and performance of your deployed models.    timeout=30
-    Wait Until Page Contains Element    ${MS_HEADER_XP}    timeout=30
+    ${version-check}=  Is RHODS Version Greater Or Equal Than  1.23.0
```

Comment:
In RHODS 1.23 there are several html changes and ods-ci is not preserving compatibility with RHODS 1.22. It would be better not to add the version check. Also, Diego is removing all the occurrences of `Is RHODS Version Greater Or Equal Than` because it's not compatible with ODH. 

We can discuss in the automation meeting how we can deal with this in the future without using that keyword

Reply:
Removed the version checks